languagetool-language-modules/sr/src/main/java/org/languagetool/language/Serbian.java

```diff
@@ -0,0 +1,166 @@
+/* LanguageTool, a natural language style checker
+ * Copyright (C) 2007 Daniel Naber (http://www.danielnaber.de)
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+ * USA
+ */
+package org.languagetool.language;
+
+import org.languagetool.JLanguageTool;
+import org.languagetool.Language;
+import org.languagetool.LanguageMaintainedState;
+import org.languagetool.databroker.ResourceDataBroker;
+import org.languagetool.rules.*;
+import org.languagetool.rules.sr.MorfologikSerbianSpellerRule;
+import org.languagetool.rules.sr.SimpleGrammarReplaceRule;
+import org.languagetool.rules.sr.SimpleStyleReplaceRule;
+import org.languagetool.synthesis.Synthesizer;
+import org.languagetool.synthesis.sr.SerbianSynthesizer;
+import org.languagetool.tagging.Tagger;
+import org.languagetool.tagging.sr.SerbianTagger;
+import org.languagetool.tokenizers.SRXSentenceTokenizer;
+import org.languagetool.tokenizers.SentenceTokenizer;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ResourceBundle;
+
+/**
+ * Support for Serbian language
+ */
+public class Serbian extends Language implements AutoCloseable {
```

Comment:
I think the `AutoCloseable` isn't needed (it's only needed for languages with ngram data).

Reply:
Removed, pendind commit.